docs/src/main/mdoc/datatypes/validated.md

```diff
@@ -523,36 +525,32 @@ Awesome! And now we also get access to all the goodness of `Applicative`, which
 
 We can now easily ask for several bits of configuration and get any and all errors returned back.
 
-```tut:silent
+```scala mdoc:silent:nest
 import cats.Apply
 import cats.data.ValidatedNec
 
-implicit val necSemigroup: Semigroup[NonEmptyChain[ConfigError]] =
+implicit def necSemigroup: Semigroup[NonEmptyChain[ConfigError]] =
   SemigroupK[NonEmptyChain].algebra[ConfigError]
 
-val config = Config(Map(("name", "cat"), ("age", "not a number"), ("houseNumber", "1234"), ("lane", "feline street")))
+val personConfig = Config(Map(("name", "cat"), ("age", "not a number"), ("houseNumber", "1234"), ("lane", "feline street")))
 
 case class Address(houseNumber: Int, street: String)
 case class Person(name: String, age: Int, address: Address)
-```
 
-Thus.
-
-```tut:book
-val personFromConfig: ValidatedNec[ConfigError, Person] =
-  Apply[ValidatedNec[ConfigError, *]].map4(config.parse[String]("name").toValidatedNec,
-                                           config.parse[Int]("age").toValidatedNec,
-                                           config.parse[Int]("house_number").toValidatedNec,
-                                           config.parse[String]("street").toValidatedNec) {
-    case (name, age, houseNumber, street) => Person(name, age, Address(houseNumber, street))
-  }
+// val personFromConfig: ValidatedNec[ConfigError, Person] =
```

Comment:
For some reason this particular example complains about forward reference extending beyond definition... and I think I already suffer from mdoc blindness as I can't figure out what exactly is wrong

All the previous instances of this problem were solved renaming some variable